lib/ObjWriter/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ endif()
 add_library(objwriter
    SHARED
    objwriter.cpp
+   objwriter.h
```

Comment:
You shouldn't need to specify headers explicitly. The compiler will pick them up from its list of include directories. If that's not working, you can tell cmake about it with `include_directories` directive.